### ISSUE TITLE
[Request] Just a little fix

### ISSUE THREAD

**Issue by askotx** [J-Rios/BashTool_ROPI_RCA#1]
Great tool, for opi boards, very useful, could I make a suggestion?

Instead of replacing full retroach.cfg in:  

https://github.com/J-Rios/BashTool_ROPI_RCA/blob/64be5e73956d7181e256169ad4245fdbf4a32604/Tool/bashFiles/main.sh#L61
https://github.com/J-Rios/BashTool_ROPI_RCA/blob/64be5e73956d7181e256169ad4245fdbf4a32604/Tool/bashFiles/main.sh#L90
https://github.com/J-Rios/BashTool_ROPI_RCA/blob/64be5e73956d7181e256169ad4245fdbf4a32604/Tool/bashFiles/main.sh#L120
https://github.com/J-Rios/BashTool_ROPI_RCA/blob/64be5e73956d7181e256169ad4245fdbf4a32604/Tool/bashFiles/main.sh#L149

This makes you lose your own settings any time you change video output, but if only we replace just audio in retroarch.cfg file in that same lines: 
```
#HDMI audio
sed -i '/audio_device = \"hw:0,0\"/d' /opt/retropie/configs/all/retroarch.cfg && echo audio_device = \"\" >> /opt/retropie/configs/all/retroarch.cfg

#Analog audio
sed -i '/audio_device = \"\"/d' /opt/retropie/configs/all/retroarch.cfg && echo audio_device = \"hw:0,0\" >> /opt/retropie/configs/all/retroarch.cfg
```
I just compared that two files, and they're almost same, just that line changes. What do you think?

**Comment by J-Rios** [comment]
Hi,

Sure, you are right about It!

I wrote the code with copy files behaviour for a fast development and because I had got in mind that the tool will be use like a first configuration tool (just configure the video-audio settings once at first boot).

I have change the code, but instead of use that lines of code that you provide (thanks for it), I make a function for changes the values of the retroarch.cfg file, for comodity and control the possibles duplicate-lines problems, if write one line that the file alredy contain It (the user run the tool twice and set the same video-audio setting).

Did not test the changes yet, but it seems should work properly...

Check the new commit for changes:
https://github.com/J-Rios/BashTool_ROPI_RCA/commit/4d6b4e17ab93da334d915c25674005541656442a
